content/tutorial/02-sveltekit/06-rendering-options/01-rendering-options-basics/README.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Rendering Options Basics
```

Comment:
```suggestion
title: Basics
```

Reply:
I think it's better to go with just "rendering options" instead.